types/jest/index.d.ts

```diff
@@ -175,67 +177,71 @@ declare namespace jest {
         stringMatching(str: string | RegExp): any;
     }
 
-    interface Matchers {
+    interface Matchers<R> {
```

Comment:
`resolves` and `rejects` return `Matchers<Promise<void>>`. How should I make this change without introducing parameters?